### PR TITLE
Geneticist Changes+Cleaned Dep ID (Modularized!)

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
This adds medsci comms to geneticists, as well as gives Geneticists easy access to medical.

## Why It's Good For The Game

Geneticists open up a new world for medical, and allows them to give humans their original body back, and also geneticists can ask medical for medicines to heal back their monkey. This gives medical access to genetics to help the players. 
As well as this, Medical now has a bridge to science, and science has a bridge to medical.

## Changelog
:cl:
add: Added medical comms to geneticists on roundstart
tweak: Changed Deputy ID temporarily to bring it in line with the other IDs
tweak: Changed Geneticist's IDs to give it the cute medical blue line, much like shaft miners

/:cl:

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
What does this do to Deputy ID’s? It’s never said in the PR body and nothing in code, unless it’s a sprite change

File: ModularTegustation/tegu_jobs/tegu_geneticist.dm [hunk 1]
```diff
@@ -0,0 +1,8 @@
+//Tegu Geneticist Edit
+
+//Adds Medsci Radio to their lineup
+/datum/outfit/job/geneticist/pre_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+    ..()
+    var/datum/job/geneticist/J = SSjob.GetJobType(jobtype)
+    if(J)
+        ears = /obj/item/radio/headset/headset_medsci
```
Comment (JohnFulpWillard) [inline ModularTegustation/tegu_jobs/tegu_geneticist.dm:8]: You're using spaces rather than tabs

Comment (Kitsunemitsu) [reply ModularTegustation/tegu_jobs/tegu_geneticist.dm:8]: I'll fix that

**Comment by Kitsunemitsu** [comment]
The deputy thing is a temporary sprite change, it looks awkward right now, I'm just returning it to the deputy ID sprite that we used to have on fulp

**Review by JohnFulpWillard — APPROVED** [review]

**Pull request merged**